jb-yoda/src/main/java/life/jukebox/YodaSubscriberClient.java

```diff
@@ -0,0 +1,36 @@
+package life.jukebox.yoda;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.SubscriptionName;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingDeque;
+
+public class YodaSubscriberClient{
+  private static final String PROJECT_ID = getEnvironmentVariable("GCP_PROJECT_ID");
+  private static final String SUBSCRIPTION_ID = getEnvironmentVariable("PUBSUB_SUB_JUKEBOX_CREATED_YODA");
+
+    private static String getEnvironmentVariable (String environmentKey) throws IllegalArgumentException {
+      if (System.getenv(environmentKey) == null) { 
+        throw new IllegalArgumentException("Environment variable " + environmentKey + " does not exist");
```

Comment:
Not sure if IllegalArgumentException is the right type of exception since an env var is not an arg. Let's just print to stderr and exit with a nonzero code instead of throwing an exception. Also change the name to getEnvironmentVariable to getEnvironmentVariableOrDie orsomething like that that explains the fatal nature of this routine.

Reply:
?

Reply:
fixed